pyproject.toml

```diff
@@ -13,6 +13,9 @@ requires-python = ">=3.8"
 dependencies = [
     'sacremoses==0.0.53',
     'sentencepiece==0.1.99',
```

Comment:
Do we really need such precise requirements? Maybe, specifying a major version of packages would be enough?

Reply:
I don't really know. I had issues with numpy when i upgraded it to use the latest version. it needed a higher version of python (python3.9 and above).